buildSrc/src/main/groovy/org/elasticsearch/gradle/test/DistroTestPlugin.java

```diff
@@ -67,35 +69,47 @@
 
     // all distributions used by distro tests. this is temporary until tests are per distribution
     private static final String PACKAGING_DISTRIBUTION = "packaging";
-    private static final String COPY_PACKAGING_TASK = "copyPackagingArchives";
+    private static final String COPY_DISTRIBUTIONS_TASK = "copyDistributions";
     private static final String IN_VM_SYSPROP = "tests.inVM";
 
-    private static Version upgradeVersion;
-    private Provider<Directory> archivesDir;
-    private TaskProvider<Copy> copyPackagingArchives;
-    private Jdk systemJdk;
-    private Jdk gradleJdk;
-
     @Override
     public void apply(Project project) {
-        project.getPluginManager().apply(JdkDownloadPlugin.class);
         project.getPluginManager().apply(DistributionDownloadPlugin.class);
-        project.getPluginManager().apply(VagrantBasePlugin.class);
-        project.getPluginManager().apply(JavaPlugin.class);
-        
-        configureVM(project);
-
-        if (upgradeVersion == null) {
-            // just read this once, since it is the same for all projects. this is safe because gradle configuration is single threaded
-            upgradeVersion = getUpgradeVersion(project);
-        }
-
-        // setup task to run inside VM
-        configureDistributions(project);
-        configureCopyPackagingTask(project);
-        configureDistroTest(project);
-        configureBatsTest(project, "oss");
-        configureBatsTest(project, "default");
+        project.getPluginManager().apply(BuildPlugin.class);
+
+        // TODO: it would be useful to also have the SYSTEM_JAVA_HOME setup in the root project, so that running from GCP only needs
```

Comment:
SYSTEM_JAVA_HOME is the java some tests use when they test the behavior of setting JAVA_HOME instead of using the bundled jdk. In 7.x, this is the java installed on the VM (thus "system") because the jdk download cannot handle java 8 since there were no oracle gpl releases of that version (we want it to be the minimum java version supported).